.github/workflows/test-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: Test branch code
+name: Test Pull Request
```

Comment:
to be precise though, it to test anything pushed to branches other than "master" or "dev"